contents/HealthChecks.md

```diff
@@ -1,3 +1,57 @@
 # Health Checks
 
-TODO
\ No newline at end of file
+Brighter provides an AspNet Core Health check for **Service Activator**
+
+## Configure Health Checks
+
+The below will configure Brighter Health checks for /health and a more detailed page for /health/detail
```

Comment:
Probably worth noting that you mean health checks at these URIs as it's not clear what the health checks give me once enabled i.e. ASP.NET health checks at these endpoints etc.